### ISSUE TITLE
Issues with setting theme programmatically

### ISSUE THREAD

**Issue by EatHeat** [JakeWharton/ActionBarSherlock#1069]
Both TitleBar and ActionBar are showing when theme is set programmatically instead of setting it in manifest. Also themes are not working properly after that. Got a black background with Theme.Sherlock.Light and a white background with Theme.Sherlock.

**Comment by JakeWharton** [comment]
Show code. You can't set a theme after the decor view has been initialized.

**Comment by EatHeat** [comment]
Its just a simple SherlockActivity calling setTheme before setContentView. The ActionBar comes below the standard pre HC TitleBar.

I'm trying to change theme of app at runtime and in NoActionBar theme, the ActionBar goes away but the TitleBar stays.

A bad workaround I found was setting the theme both in manifest and programmatically. With a few tweaks like setting the background colour again, I was able to make it work. It had a few glitches though like in the Theme.Sherlock theme, the blue line below the ActionBar was not coming under the ActionBar menus.

Also I can't switch to fullscreen theme when setting the theme in both manifest and programmatically. The manifest theme seems to have a higher priority.

Hope you fix it.
